src/api/Nhs.Appointments.Core/IAvailabilityStore.cs

```diff
@@ -5,6 +5,9 @@ public interface IAvailabilityStore
     Task<IEnumerable<SessionInstance>> GetSessions(string site, DateOnly from, DateOnly to);
    Task ApplyAvailabilityTemplate(string site, DateOnly date, Session[] sessions, ApplyAvailabilityMode mode,
         Session sessionToEdit = null);
-    Task<IEnumerable<DailyAvailability>> GetDailyAvailability(string site, DateOnly from, DateOnly to);
-    Task<SessionInstance> CancelSession(string site, DateOnly date, Session session);
+   Task<IEnumerable<DailyAvailability>> GetDailyAvailability(string site, DateOnly from, DateOnly to);
+   Task<SessionInstance> CancelSession(string site, DateOnly date, Session session);
+
+   Task<bool> SiteSupportsService(string siteId, string service,
+       List<string> dailyAvailabilityIds);
```

Comment:
Could I just suggest some minor naming tweaks here? 

I know since you're passing dates in it's kind of implicit that you're checking a range, but that param is called Id so this method actually looks like a global on/off sort of thing. I think something like `SiteOffersServiceDuringPeriod` would convey the content more clearly.
Likewise, instead of `dailyAvailabilityIds`, I'd call it something more like `DatesInPeriod`. The fact that the Id of our cosmos documents is the date string is a facet of our implementation, and by naming this param Ids we bake that into this method. I'd rather describe them a dates because that's clearer to the caller and agnostic for the future